File: src/main/java/com/lunatech/testing/ad/entities/Runway.java
package com.lunatech.testing.ad.entities;

public class Runway extends BaseEntity{
	/**Runways csv contains:
	 * id,"airport_ref","airport_ident","length_ft","width_ft",
	 * "surface","lighted","closed","le_ident","le_latitude_deg",
	 * "le_longitude_deg","le_elevation_ft","le_heading_degT",
	 * "le_displaced_threshold_ft","he_ident","he_latitude_deg",
	 * "he_longitude_deg","he_elevation_ft","he_heading_degT","he_displaced_threshold_ft",
	 */
	
	private Long airport_ref;
	private String surface;
	private String le_ident;
	
	public Long getAirport_ref() {
		return airport_ref;
	}
	public void setAirport_ref(Long airport_ref) {
		this.airport_ref = airport_ref;
	}
	public String getSurface() {
		return surface;
	}
	public void setSurface(String surface) {
		this.surface = surface;
	}
	public String getLe_ident() {
		return le_ident;
	}
	public void setLe_ident(String le_ident) {
		this.le_ident = le_ident;
	}
	@Override
	public String toString() {
		return "Runway [airport_ref=" + airport_ref + ", surface=" + surface + ", le_ident=" + le_ident + ", getId()="
				+ getId() + "]";
	}
	
}

File: app/models/Runway.scala
package models

case class Runway (id: String, airport_ref: String, airport_ident: String, length_ft: Int, width_ft: Int,
                  surface: String, lighted: Int, closed: Int, le_ident: String, le_latitude_deg: Float,
                  le_longitude_deg: Float, le_elevation_ft: Int, le_heading_degT: Float, le_displaced_threshold_ft: Int,
                  he_ident: String, he_latitude_deg: Float, he_longitude_deg: Float, he_elevation_ft: Int,
                  he_heading_degT: Int, he_displaced_threshold_ft: Int)